src/lib.rs

```diff
@@ -92,76 +126,23 @@ pub fn hash(data: &[u8]) -> [u8; 32] {
     hashing_impl::sha3_256(data)
 }
 
-/// Represents a set of public keys, consisting of a public signature key and a public
-/// encryption key.
+/// Use `gen_encrypt_keypair()` to generate a public and private key pair.
```

Comment:
I think this doc comment should first state what `PublicEncryptKey` is rather than how to create one.  Also, we're using the term "secret" rather than "private" when describing crypto keys now.